src/main/java/seedu/address/model/ReadOnlyBookingBook.java

```diff
@@ -0,0 +1,9 @@
+package seedu.address.model;
+
+import seedu.address.model.booking.Booking;
+
+public interface ReadOnlyBookingBook {
```

Comment:
I think this interface is supposed to only return a list of bookings. Can try looking at ReadOnlyAddressBook and find its usages to see how it's used and we can just follow! If I am not wrong it's used with storage. Other than this, LGTM!